### DISCUSSION TITLE
Split videos into chunks for faster loading

### DISCUSSION THREAD

**Discussion by pipe01** [ente-io/ente#4534] [Enhancements]
It would be nice if ente automatically split big videos into multiple chunks in the backend to allow for watching long videos without having to wait for the whole thing to be downloaded and decrypted.

**Comment by Fricsion** [comment]
In my humble opinion, this is an indispensable feature considering the way Ente works. 

A normal 4K60FPS 4min30sec video takes 1 minute to download the file + 1 minute to decrypt on my iPhone 13. 

So there's 2 whole minutes of waiting. 

I'd be okay because I fully understand what and why it needs to do. But my family don't. They simply think Ente is tad slower for what it does than Apple/Google photos...
